src/shared-styles/_font.scss

```diff
@@ -1,3 +1,4 @@
+$letterspacing-label: 0.5px !default;
```

Comment:
Doesn't need to be configurable. Move to the `_label-typography` mixin as a private variable.

Should also rename to `$_letter-spacing` at the same time.